states.inc.php

```diff
@@ -295,13 +295,13 @@
     'possibleactions' => ['actCirclePlant', 'actRestart', 'actPassOptionalAction'],
   ],
 
-  ST_STIR_WATER_TANKS => [
-    'name' => 'stirWaterTanks',
+  ST_CIRCLE_SINGLE_LINKED => [
+    'name' => 'circleSingleLinked',
     'descriptionmyturn' => clienttranslate('${you} may choose to stir the attached water tank'),
```

Comment:
you need to handle the different descriptions for the different cases here.
Instead of just using constants named CIRCLE_XXXX, I would maybe suggest to add the scenario number somewhere in the constant name to make it easier to see which constant is used in which scenario.
It might also be useful for scenario specific actions to make it easier to tell them apart as well.

Reply:
Done